src/crypto/file-decrypt.ts

```diff
@@ -1,18 +1,19 @@
 import { SkynetClient } from 'skynet-js';
 import CryptoJS from 'crypto-js';
 import { EncryptedFileReference } from '../models/encryption';
-import { DEFAULT_DOMAIN, MAX_AXIOS_RETRIES } from '../config';
+import { FileDecrypt } from './crypto';
+import { getEndpoint } from '../portals';
+import { MAX_AXIOS_RETRIES } from '../config';
 import axios from 'axios';
 import axiosRetry from 'axios-retry';
 import { ChunkResolver } from './chunk-resolver';
-import { FileDecrypt } from './crypto';
 
 export default class AESFileDecrypt implements FileDecrypt {
   private encryptedFile: EncryptedFileReference;
   private encryptionKey: string;
   private chunkResolver: ChunkResolver;
 
-  skynetClient = new SkynetClient(DEFAULT_DOMAIN);
+  skynetClient = new SkynetClient(getEndpoint());
```

Comment:
Here we should use the `getCurrentPortal()` as it works as the upload. 

During the decrypt, skynetClient is used to convert something like `sia://_AVpGXKGy-aQ61fXh8DP62rKEfMWxEUFKyRsKCX98X8Ocg` to a portal link like: 

https://siasky.net/_AVpGXKGy-aQ61fXh8DP62rKEfMWxEUFKyRsKCX98X8Ocg
https://skyportal.xyz/_AVpGXKGy-aQ61fXh8DP62rKEfMWxEUFKyRsKCX98X8Ocg
https://skydrain.net/_AVpGXKGy-aQ61fXh8DP62rKEfMWxEUFKyRsKCX98X8Ocg

Reply:
I checked and when downloading from https://skyportal.xyz it returns empty files